erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py

```diff
@@ -226,7 +226,7 @@ def new_bank_transaction(transaction):
 	try:
 		tags += transaction["category"]
 		tags += ["Plaid Cat. {}".format(transaction["category_id"])]
-	except KeyError:
+	except Exception:
```

Comment:
Instead of simply ignoring all the exceptions can you just add the one that you got and can be ignored

Reply:
I have made the change. still, because the tags are not mandatory i question the importance to make this a failing error. i would think the creation of the document is more important than the labels in this case. 
perhaps make the other exceptions an assignment to the admin but not failing creation ?